src/infrastructure/figma/transformers/figma-file-meta-transformer.ts

```diff
@@ -19,6 +23,14 @@ type TransformFileMetaToAtlassianDesignParams = {
 	readonly fileMetaResponse: GetFileMetaResponse;
 };
 
+const transformFigmaUserToAtlassianProviderUser = (
```

Comment:
**suggestion (non-blocking):** It is okay to map a single field manually within `transformFileMetaToAtlassianDesign` without calling a separate function (`transformFigmaUserToAtlassianProviderUser`).

Please, pay attention that if you would like to encapsulate the mapping in a function, it should be moved out of this module as the mapping is required in 3 places:

- `transformFileMetaToAtlassianDesign`
- `transformFileToAtlassianDesign`
- `transformNodeToAtlassianDesign`

Reply:
Updated to use a dedicated user transformer. Tests also updated to check only id is returned.